docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -167,8 +167,8 @@ Create your own key with OpenAI <a href="https://beta.openai.com/account/api-key
   "completions": {
     "provider": "openai",
     "chatModel": "gpt-4", // Or any other model you would like to use
-    "fastChatModel": "gpt-35-turbo", // Or any other model you would like to use
-    "completionModel": "gpt-35-turbo", // Or any other model you would like to use
+    "fastChatModel": "gpt-3.5-turbo", // Or any other model you would like to use
+    "completionModel": "gpt-3.5-turbo", // Or any other model you would like to use
```

Comment:
```suggestion
    "completionModel": "gpt-3.5-turbo-instruct", // Or any other model you would like to use
```
gpt-3.5-turbo after 0301 dropped support for the /completions endpoint which autocomplete was using

Reply:
Oh, TIL!